docs/api/en/objects/InstancedMesh.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8" />
+		<base href="../../../" />
+		<script src="list.js"></script>
+		<script src="page.js"></script>
+		<link type="text/css" rel="stylesheet" href="page.css" />
+	</head>
+	<body>
+		[page:Mesh] &rarr;
+
+		<h1>[name]</h1>
+
+		<p class="desc">
+		A special version of [page:Mesh] with instanced rendering support. Use [name] if you have to render a large amount of
```

Comment:
Nit: Here, and throughout the rest of the page, I would replace the word "amount" with "number". The latter is preferred for objects that can be counted discretely. (https://www.dailywritingtips.com/amount-vs-number/)

Reply:
Ok, let me make a new PR with your suggested change 👍